diff -up iotop-0.3.2/iotop/ui.py.sigpipe iotop-0.3.2/iotop/ui.py
--- iotop-0.3.2/iotop/ui.py.sigpipe	2016-11-02 23:19:48.716444619 +0100
+++ iotop-0.3.2/iotop/ui.py	2016-11-02 23:21:05.811856634 +0100
@@ -6,6 +6,7 @@ import optparse
 import os
 import pwd
 import select
+import signal
 import struct
 import sys
 import time
@@ -440,6 +441,8 @@ class IOTopUI(object):
             self.win.refresh()
 
 def run_iotop_window(win, options):
+    if options.batch:
+        signal.signal(signal.SIGPIPE, signal.SIG_DFL)
     taskstats_connection = TaskStatsNetlink(options)
     process_list = ProcessList(taskstats_connection, options)
     ui = IOTopUI(win, process_list, options)
