diff -up iotop-0.3.2/iotop/ioprio.py.ppcprio iotop-0.3.2/iotop/ioprio.py
--- iotop-0.3.2/iotop/ioprio.py.ppcprio	2013-09-13 17:49:34.695139596 +0200
+++ iotop-0.3.2/iotop/ioprio.py	2013-09-13 17:50:05.157831162 +0200
@@ -13,7 +13,7 @@ IOPRIO_GET_ARCH_SYSCALL = [
     ('alpha',       '*',  443),
     ('i*86',        '*',  290),
     ('ia64*',       '*', 1275),
-    ('powerpc*',    '*',  274),
+    ('ppc*',        '*',  274),
     ('s390*',       '*',  283),
     ('sparc*',      '*',  218),
     ('sh*',         '*',  289),
@@ -25,7 +25,7 @@ IOPRIO_SET_ARCH_SYSCALL = [
     ('alpha',       '*',  442),
     ('i*86',        '*',  289),
     ('ia64*',       '*', 1274),
-    ('powerpc*',    '*',  273),
+    ('ppc*',        '*',  273),
     ('s390*',       '*',  282),
     ('sparc*',      '*',  196),
     ('sh*',         '*',  288),
