diff -up iotop-0.3.2/iotop/ui.py.batchprintutf8 iotop-0.3.2/iotop/ui.py
--- iotop-0.3.2/iotop/ui.py.batchprintutf8	2016-11-02 14:49:51.560485909 +0100
+++ iotop-0.3.2/iotop/ui.py	2016-11-02 14:49:51.586486103 +0100
@@ -402,7 +402,7 @@ class IOTopUI(object):
                 if self.options.quiet <= int(first_time):
                     print ''.join(titles)
             for l in lines:
-                print l
+                print l.encode('utf-8')
             sys.stdout.flush()
         else:
             self.win.erase()
