--- a/iotop/ui.py
+++ b/iotop/ui.py
@@ -512,7 +512,10 @@ option, a to toggle the --accumulated option, q to quit, any other key to force
 a refresh.''' % sys.argv[0]
 
 def main():
-    locale.setlocale(locale.LC_ALL, '')
+    try:
+        locale.setlocale(locale.LC_ALL, '')
+    except locale.Error:
+        print 'unable to set locale, falling back to the default locale'
     parser = optparse.OptionParser(usage=USAGE, version='iotop ' + VERSION)
     parser.add_option('-o', '--only', action='store_true',
                       dest='only', default=False,
