diff -urN iotop-0.6.orig/iotop/ioprio.py iotop-0.6/iotop/ioprio.py
--- iotop-0.6.orig/iotop/ioprio.py	2013-05-27 00:44:18.000000003 +0200
+++ iotop-0.6/iotop/ioprio.py	2018-08-01 15:12:59.460788725 +0200
@@ -27,6 +27,7 @@
 # 'x86_64' but it will use the i386 syscall number, that's why we consider both
 # the architecture name and the word size.
 IOPRIO_GET_ARCH_SYSCALL = [
+    ('aarch64',     '*',   31),
     ('alpha',       '*',  443),
     ('arm*',        '*',  315),
     ('i*86',        '*',  290),
@@ -41,6 +42,7 @@
 ]
 
 IOPRIO_SET_ARCH_SYSCALL = [
+    ('aarch64',     '*',   30),
     ('alpha',       '*',  442),
     ('arm*',        '*',  314),
     ('i*86',        '*',  289),
